### PR TITLE
chore(ecosystem): Update page

### PR DESCRIPTION
**Pull request by pilab-jinsu**
**What changed? Why?**
Updates the ecosystem page
- Update ecosystem.json
- Add a logo

**Notes to reviewers**
Hello,
Thank you for your feedback on the previous [PR](https://github.com/base-org/web/pull/1310#issuecomment-2525057968).
I’ve updated the service so that after clicking "start here," is automatically set to cbBTC, as requested.
Looking forward to your review. Please let me know if further changes are needed. 🙏

**How has it been tested?**
localhost

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Comment by wbnns** [comment]
@pilab-jinsu Thank you for the update and new PR. The language when depositing does not match that of cbBTC which is seemingly confusing since WBTC and/or BTCB wouldn't be what are getting deposited for CBBTC. Also, could you please reply and share the contract addresses for BTCUSD for all networks it is deployed on, as well as the custody addresses for the corresponding assets that have been supplied to mint it?
2024-12-28 at 10 36 02@2x

**Comment by pilab-jinsu** [comment]
@wbnns Thank you for the detailed reply.
To address the issuee that you mentioned, we updated the phrase of the page that you mentioned to reduce the confusion when depositing does not match that of cbBTC.

For the contract addresses of BtcUSD, here’s the updated list of contract addresses for BtcUSD:
1. BtcUSD address
    - Bifrost Network: 0x6906ccda405926fc3f04240187dd4fad5df6d555
    - Base: 0xe4b20925d9e9a62f1e492e15a81dc0de62804dd4
    - CORE: 0xad0c524ce19cea03654dc377da7bac52c56edd10
1. Custudy (Vault)
    - Bifrost Network: 0xD85EB87caB9041ad00764b95796702b1104F42D7
    - Base: 0x4F7aB59b5AC112970F5dD66D8a7ac505c8E5e08B
    - CORE: 0x872b347cd764d46c127ffefbcaB605FFF3f3a48C
  
Please let me know if you have any questions.

**Comment by wbnns** [comment]
@pilab-jinsu 

Thanks for the update! Can you share the BtcUSD and custody addresses for the other networks you support on the site, as well? I see three pairs above but on the user interface there are 10 networks/assets that can be deposited from.

It would be helpful to have a link showing where everything is backed 1:1, verifiable onchain.

**Comment by pilab-jinsu** [comment]
@wbnns Thank you for the update!
While our service supports approximately 10 depositable pairs, BTCUSD is only supported on the following 4 networks.
The custody addresses for these networks are as follows:

Bifrost Network: 0xD85EB87caB9041ad00764b95796702b1104F42D7
Base: 0x4F7aB59b5AC112970F5dD66D8a7ac505c8E5e08B
CORE: 0x872b347cd764d46c127ffefbcaB605FFF3f3a48C
BNB: 0x78ae4c0FD4f02CA79A2d8738d3369A4Bc5D4E323

**Comment by wbnns** [comment]
@pilab-jinsu Got it, thanks! What's the best way to see that custody = supply between all of the networks?

**Comment by pilab-jinsu** [comment]
@wbnns Thank you for the update!

Base: [0xe4b20925d9e9a62f1e492e15a81dc0de62804dd4](https://basescan.org/token/0xe4b20925d9e9a62f1e492e15a81dc0de62804dd4)
BNB: [0x4f2c996248ed9592e64b59f781a15deb1e2b0d4c](https://bscscan.com/token/0x4f2c996248ed9592e64b59f781a15deb1e2b0d4c)
CORE: [0xad0c524ce19cea03654dc377da7bac52c56edd10](https://scan.coredao.org/token/0xad0c524ce19cea03654dc377da7bac52c56edd10)

The sum of the total supply of these three networks is equal to the BTCUSD balance held in the Bifrost Vault at:
Bifrost Network: [0xD85EB87caB9041ad00764b95796702b1104F42D7](https://explorer.mainnet.bifrostnetwork.com/address/0xD85EB87caB9041ad00764b95796702b1104F42D7)

Additionally, the Bifrost network Vault holds slightly more BTCUSD since it collects bridge fees.

**Comment by wbnns** [comment]
@pilab-jinsu Thanks for the update. When users deposit cbBTC or another accepted asset, this bridges it to Bifrost, is that correct? Can they bridge to/from other networks? If they are earning yield on Bifrost, should the custody amount on Base and other networks be greater, since they would be unwrapping BtcUSD to get back to cbBTC which would also include the extra amount that is the accumulated yield?

**Comment by pilab-jinsu** [comment]
@wbnns Thank you for the update!

When users deposit cbBTC or other supported assets, they are bridged to Bifrost. BtcUSD can also be bridged between supported networks.

BTCFi is a decentralized crosschain Bitcoin asset management platform offering native Bitcoin collateralization to mint a Bitcoin-backed stablecoin, BtcUSD.
When users get back to cbBTC, only the initially locked collateral is returned, so custody balances on Base and other networks do not increase from yield accumulation.

Hope this helps! Let me know if you have any further questions.

**Comment by wbnns** [comment]
@pilab-jinsu 

Thank you for the additional information. Can you please share more details on how the yield is generated?

**Comment by pilab-jinsu** [comment]
@wbnns Thank you for the update!
BTCFi is a Bitcoin staking service that enables users to mint BtcUSD through a CDP. Users can mint BtcUSD up to 50% of staked amount.
Once users mint BtcUSD, they have multiple ways to earn yield on other DeFi Services. Currently, BTCFi offers yield 
  
  opportunities through:
- BiFi – Deposit BtcUSD and earn yield at the given rate. Funds can be withdrawn at any time.
- Everdex – Provide liquidity to the BtcUSD-USDC stablecoin pool and earn a share of the trading fees. Additionally, 
providing liquidity on LP pools can further boost rewards.

**Comment by wbnns** [comment]
@pilab-jinsu Thanks for following up! Apologies, it still seems a bit unclear:

> BiFi – Deposit BtcUSD and earn yield at the given rate. Funds can be withdrawn at any time.

Where does the yield specifically come from?

**Comment by pilab-jinsu** [comment]
@wbnns Hello, thank you for the reply.
BTCFi provides service that is integrated with other DeFi services such as lending and Dex where users can utilize minted BtcUSD.
If a user chooses to use those services then they can either supply BtcUSD on lending service or provide liquidity on DEX.
Currently BTCFi provides an easy way for users to supply minted BtcUSD to lending service and a DEX service.
Basically yields are generated on other DeFi services.

**Comment by wbnns** [comment]
@pilab-jinsu 

Thanks for sharing more info. RE:

> Basically yields are generated on other DeFi services.

Please link to the DeFi services and the relevant components that create the yield for the BtcUSD.

**Comment by pilab-jinsu** [comment]
@wbnns 
Below are the links to the URL that users can generate yields.
Everdex (Crosschain Dex): https://www.everdex.app/pool
BtcUSD / USDC Pool: https://www.everdex.app/pool/0x840cf4522ed96cbbeb0924672ea170456eea3a4c
BtcUSD / BFC Pool: https://www.everdex.app/pool/0x908b3cb9f8e6441b2b1844a6d4f1ac4707bd1483
BiFi (Crosschain Ledning): https://crosschain.bifi.finance/

**Comment by wbnns** [comment]
@pilab-jinsu 

Thanks! Apologies, I meant please share more details on how each pool creates the advertised yield. For example, if one deposits cbBTC on Base, how is the yield generated when they withdraw, say 1month later. It's a bit unclear.

**Comment by pilab-jinsu** [comment]
@wbnns 
Here is a simplified explanation of how BTCFi works and how users generate yield when they withdraw:
BTCFi is a BTC staking service that allows users to mint a Bitcoin-backed stablecoin called **BtcUSD** by staking their BTC.
Once users mint BtcUSD, they can choose to use it in different ways to earn yield — mainly through **lending services** or **DEXs (decentralized exchanges)**.
1. **Lending Option**:
    - If a user chooses the lending option, their BtcUSD will be supplied to a lending protocol.
    - The lending protocol offers a **supply APY (Annual Percentage Yield)**, and users earn interest based on this rate.
        - For example: If a user supplies **$100 of BtcUSD** to a lending protocol offering **12% APY**, the user would earn about **$1 in one month**.
2. **DEX Option**:
    - If a user chooses the DEX option, they will provide their BtcUSD to a liquidity pool.
    - Whenever someone makes a **swap (trade)** using that pool, the user earns **a portion of the swap fees**, which is reflected as yield.
        - For example: If a user provides **$100 of BtcUSD** to a DEX pool and the average yield is around **12% APY**, they would similarly earn around **$1 per month**.
  
In summary:
BTCFi allows users to earn passive income on their staked BTC by minting BtcUSD and utilizing it in yield-generating protocols like lending platforms or DEXs.

**Comment by wbnns** [comment]
@pilab-jinsu 

Hello, thank you. Apologies, we're unable to add BTCFi at this time. Once more details are available as to exactly how the supply APY and interest yields are derived, that can be verified onchain, please open a new PR. The concept is clear but the specific details are missing. 🙏
